R/AllGenerics.R

```diff
@@ -143,3 +143,9 @@ setGeneric("filterEmptySpectra", function (object, ...) standardGeneric("filterE
 setGeneric("bpi", function(object, ...) standardGeneric("bpi"))
 setGeneric("spectrapply", function(object, ...) standardGeneric("spectrapply"))
 setGeneric("splitByFile", function(object, f, ...) standardGeneric("splitByFile"))
+
+## Chromatogram class:
+setGeneric("productMz", function(object, value) standardGeneric("productMz"))
+setGeneric("aggregationFun", function(object, ...)
```

Comment:
Done in https://github.com/lgatto/MSnbase/commit/978e686f6920061d8c705220cdd6eb21729a3882